.depcheckrc.yml

```diff
@@ -1,16 +1,6 @@
 ignores:
-  [
-    '@types/chai',
-    'chai',
-    'depcheck',
-    'eslint*',
-    'gulp-typescript',
-    'mocha',
-    'nyc',
-    'prettier',
-    'tslib',
-    'typescript',
-    'typedoc-plugin-external-module-map',
-    'typedoc-plugin-nojekyll',
-    'utility-types',
-  ]
+  - gulp-typescript
+  # Unused and could be removed, but leaving because it could be helpful
```

Comment:
helpful for what usecase?

Reply:
Helpful for creating type definitions. We don't currently do anything complex with our types, but we definitely could.